activity.py

```diff
@@ -16,51 +31,52 @@
 from sugar3.activity.widgets import StopButton
 from game_engine import Game
 
+BUTTON_LABELS = ("Easy", "Medium", "Hard")
+MENU_BACKGROUND_PATH = 'backgrounds/menu_background.png'
+GAME_BACKGROUND_PATH = 'backgrounds/background{}.png'
+MAP_PATH = 'map{}.json'
+BOX_SPACING = 6
+
 
 class TextdungeonActivity(activity.Activity):
     def __init__(self, handle):
-        activity.Activity.__init__(self, handle)
+        super().__init__(handle)
         self.max_participants = 1
 
-        self.build_toolbar()
-
+        self.create_toolbar()
         self.create_menu()
 
     def create_menu(self):
-        # Loading menu background image and creating pixbuf
-        menu_pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_scale(
-            filename='backgrounds/menu_background.jpg',
-            width=Gdk.Screen.width(),
-            height=Gdk.Screen.height(),
-            preserve_aspect_ratio=True)
-
-        # Creating image and setting pixbuf
-        menu_img = Gtk.Image.new_from_pixbuf(menu_pixbuf)
-
-        # Creating overlay and packing image
-        menu_overlay = Gtk.Overlay()
-        menu_overlay.add(menu_img)
-
-        # Create a vertical box to stack widgets
-        self.menu_vbox = Gtk.Box(
-            orientation=Gtk.Orientation.VERTICAL, spacing=6)
-        menu_overlay.add_overlay(self.menu_vbox)
-        self.menu_vbox.set_halign(Gtk.Align.CENTER)
-        self.menu_vbox.set_valign(Gtk.Align.CENTER)
-
-        # Create three buttons
-        for i in range(1, 4):
-            button = Gtk.Button(label="Button " + str(i))
-            if i == 1:
-                button = Gtk.Button(label="Easy")
-            elif i == 2:
-                button = Gtk.Button(label="Medium")
-            else:
-                button = Gtk.Button(label="Hard")
+        menu_overlay = self.create_overlay_from_image(MENU_BACKGROUND_PATH)
+
+        button_css = """
+        button {
+            background-color: #0B0B45;
+            color: white;
+            font: 20px Roboto, sans-serif;
+            border-radius: 10px;
+            padding: 5px 10px;
+        }
+        button:hover {
+            background-color: #23395d;
+        }
+        """
+        # Create a CssProvider
+        css_provider = Gtk.CssProvider()
+        css_provider.load_from_data(button_css.encode())
```

Comment:
If this is the only place where we'll need to load a css file then I'm fine with doing it this way else, it'll be better to create a folder for the css files and load with `Gtk.CssProvider().load_from_file(file_name)`.

Reply:
Yep, definitely.